### PR TITLE
Fix the count sort

### PR DESCRIPTION
**Pull request by wparslan**

### CODE REVIEW

**Comment by wparslan** [comment]
@darwinz Kindly merge it, it is my first Hacktoberfest ever

File: Sorting/CountSort.php [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
- * @return mixed
+ * Sorts an array of integers using the Counting Sort algorithm.
+ *
+ * @param array $array The input array to be sorted.
+ * @param int $min The minimum value in the input array.
+ * @param int $max The maximum value in the input array.
+ * @return array The sorted array.
  */
-function countSort($array, $min, $max)
+function countSort( $array, $min, $max )
```
Comment (darwinz) [inline Sorting/CountSort.php:11]: We've tried to stick with the PSR-12 style guide.  Please refer to the guide [here](https://xoops.gitbook.io/xoops-modules-cookbook/coding-standards/standards/styleguide).  These spaces surrounding arguments and parameters aren't necessary

**Review by darwinz — CHANGES_REQUESTED** [review]
This is a good change, but just take out the extra spaces that you've added inside the parentheses.  You will also need to resolve conflicts in your branch, which you can do by rebasing.

File: Sorting/CountSort.php (hunk 1 above)
Comment (wparslan) [reply Sorting/CountSort.php]: This is noted, PSR-12 Standard is followed here.

**Comment by wparslan** [comment]
Lets merge it

File: Sorting/CountSort.php [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
- * @return mixed
+ * Sorts an array of integers using the Counting Sort algorithm.
+ *
+ * @param array $array The input array to be sorted.
+ * @param int $min The minimum value in the input array.
+ * @param int $max The maximum value in the input array.
+ * @return array The sorted array.
  */
-function countSort($array)
+function countSort(array $array, int $min, int $max)
```
Comment (darwinz) [inline Sorting/CountSort.php:11]: This was changed in a recently merged PR.  These arguments are no longer needed - the min and max are calculated within this function now.

File: Sorting/CountSort.php [hunk 3]
```diff
@@ -1,30 +1,40 @@
...
 {
-    $count = array();
-    $min = min($array);
-    $max = max($array);
```
Comment (darwinz) [inline Sorting/CountSort.php:11]: This change is not needed either.

**Review by darwinz — CHANGES_REQUESTED** [review]
Please revert the change you made to the countSort function

**Pull request closed without merging**